### PR TITLE
Automatically open shops & stalls

### PR DESCRIPTION
**Pull request by ecnepsnai**
- Added a new option under misc to automatically open shops and stalls
 - Option is disabled by default
 - When enabled, shops & stalls are automatically opened right after being placed on the map

### CODE REVIEW

File: data/language/english_uk.txt [hunk 1]
```diff
@@ -3924,6 +3924,8 @@ STR_5582    :Trap mouse cursor in window
 STR_5583    :{COMMA1DP16}ms{POWERNEGATIVEONE}
 STR_5584    :SI
 STR_5585    :{SMALLFONT}{BLACK}Unlocks ride operation limits, allowing for things like {VELOCITY} lift hills
+STR_5586    :Automatically open shops and stalls
+STR_5587    :{SMALLFONT}{BLACK}When enabled, shops and stalls will be automatically opened after placing then in the map
```
Comment (IntelOrca) [inline data/language/english_uk.txt]: Typo, "them"

File: src/windows/ride_construction.c [hunk 2]
```diff
@@ -585,6 +585,12 @@ static void window_ride_construction_close(rct_window *w)
 
 	uint8 rideIndex = _currentRideIndex;
 	if (sub_6CAF80(rideIndex, &mapElement) || network_get_mode() == NETWORK_MODE_CLIENT) {
+
+		rct_ride *ride = GET_RIDE(rideIndex);
+		if (ride->mode == RIDE_MODE_SHOP_STALL && gConfigGeneral.auto_open_shops) {
+			ride->status = 1;
```
Comment (IntelOrca) [inline src/windows/ride_construction.c]: Use ```RIDE_STATUS_OPEN```

File: data/language/english_uk.txt (hunk 1 above)
Comment (Gymnasiast) [inline data/language/english_uk.txt]: Also, "on the map", not "in the map".

**Comment by Gymnasiast** [comment]
Nice idea!

File: data/language/english_uk.txt (hunk 1 above)
Comment (IntelOrca) [inline data/language/english_uk.txt]: Map seems a bit too technical anyway. Perhaps it should just say "after building them".

Comment (Gymnasiast) [inline data/language/english_uk.txt]: +1

**Comment by ecnepsnai** [comment]
@IntelOrca Ted, I've addressed your comments in this next commit.

**Pull request merged**